doc/osh-help.md

```diff
@@ -54,14 +54,16 @@ Otherwise:
 - `bin/osh` runs `~/.config/oil/oshrc`
 - `bin/oil` runs `~/.config/oil/oilrc`
 
-Pass --rcfile /dev/null to disable this behavior.
+Pass --rcfile /dev/null or --norc to disable this behavior. The latter also disables --rcdir.
 
 If the --rcdir flag is specified, all files in that folder will be executed on startup.
 Othewise:
 
 - `bin/osh` runs everything in `~/.config/oil/oshrc.d/`
 - `bin/oil` runs everything in `~/.config/oil/oilrc.d/`
 
+Pass --rcdir /dev/null or --norc to disable this behavior. The latter also disables --rcfile.
```

Comment:
How about collapsing many of these sentences into one?

Like

> Pass --norc to disable both --rcfile and --rcdir.

Reply:
Yeah it's a bit cleaner that way. But I added the sentence under both the `Pass --rcfile ...` and `Pass --rcdir ...` so that people will not miss it. Or do you think it's enough to only have it in one place?